Documentation/workflow/DevelopmentTips.md

```diff
@@ -462,3 +462,48 @@ copying `.nupkg` files to the `library-packs` directory of a given
 
 The `library-packs` directory is simply an implicit NuGet feed that is
 automatically picked up by the .NET SDK.
+
+## Enabling Mono Logging
+
+Since [6e58ce4][6e58ce4], logging from Mono is no longer enabled by
+default. You can set the `debug.mono.log` system property to answer
+questions like: Is AOT working? Is the Mono Interpreter enabled?
+
+If you wanted to enable logging for AOT, for example:
+
+```bash
+$ adb shell setprop debug.mono.log mono_log_level=debug,mono_log_mask=aot
```

Comment:
There's some documentation about the log environment variables here https://www.mono-project.com/docs/advanced/runtime/logging-runtime-events/, it's not full however. It would be good to mention the relevant bits here. Mono manpage has more information:

```
You can use the MONO_LOG_LEVEL and MONO_LOG_MASK environment variables to get verbose debugging output about the execution of your application within Mono.                                                                                                           

       The  MONO_LOG_LEVEL  environment  variable  if  set, the logging level is changed to the set value. Possible values are "error", "critical", "warning", "message", "info", "debug". The default value is "error". Messages with a logging level greater then or
       equal to the log level will be printed to stdout/stderr.                                                                                                                                                                                                              

       Use "info" to track the dynamic loading of assemblies.
                                                                                                                                                                                                                                                                             
       Use the MONO_LOG_MASK environment variable to limit the extent of the messages you get: If set, the log mask is changed to the set value. Possible values are "asm" (assembly loader), "type", "dll" (native library loader), "gc" (garbage  collector),  "cfg"
       (config  file  loader),  "aot" (precompiler), "security" (e.g. Moonlight CoreCLR support), "threadpool" (thread pool generic), "io-selector" (async socket operations), "io-layer" (I/O layer - processes, files, sockets, events, semaphores, mutexes and han‐
       dles), "io-layer-process", "io-layer-file", "io-layer-socket", "io-layer-event", "io-layer-semaphore", "io-layer-mutex", "io-layer-handle" and "all".  The default value is "all". Changing the mask value allows you to display only messages  for  a  certain
       component. You can use multiple masks by comma separating them. For example to see config file messages and assembly loader messages set you mask to "asm,cfg".
```

Reply:
I added a link to these docs:

> See the [Mono documentation][mono-logging] for more information about `MONO_LOG_LEVEL` and `MONO_LOG_MASK`.

[mono-logging]: https://www.mono-project.com/docs/advanced/runtime/logging-runtime-events/

Reply:
I opened a [PR to update](url) the Mono docs